product_seasonality/models/seasonal_config_line.py

```diff
@@ -59,3 +60,15 @@ def find_for_product(self, prod, config=None):
         if config:
             domain.append(("seasonal_config_id", "=", config.id))
         return self.search(domain)
+
+    def _compute_display_name(self):
+        for rec in self:
+            rec.display_name = rec._name_get()
+
+    def _name_get(self):
+        parts = [
+            f"[{self.seasonal_config_id.display_name}]",
```

Comment:
As seasonal config as no display name defined, I would have name here, no?
```suggestion
            f"[{self.seasonal_config_id.name}]",
```

Reply:
display_name is the safest in theory. All models have it no matter if you have name or not. It normally falls back to `name`.
That's why I did this fix https://github.com/OCA/search-engine/pull/89 because it's more reliable.

Reply:
Ok, then if we add a custom display name later, it will continue to works here: :+1: